lms/envs/common.py

```diff
@@ -2758,6 +2759,7 @@
 MOBILE_APP_USER_AGENT_REGEXES = [
     r'edX/org.edx.mobile',
 ]
+MOBILE_APP_VERSION_REGEX = r'[0-9]+.[0-9]+.[0-9]+(.[0-9a-zA-Z]*)?'  # X.X.X.Y where X is a number and Y is alphanumeric
```

Comment:
My understanding is that for future releases we are planning to follow the Semantic Versioning spec. @aleffert Is that assumption correct?

http://semver.org/#spec-item-2